package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unpm",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
Its usually better to bump the version on release (using `npm version patch|minor|major`, this is nice because it will create a tag in git for the version, as well as a commit for that tag, making it easy to checkout specific versions.

If you can revert this change, I'll do this after merge